process.sh

```diff
@@ -35,9 +35,8 @@ main() {
     then
         echo 'CI: Skip push to AWS and GCS'
     else
-        TILES_BASE='vanderbilt.images'
-        aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --exclude "$OUTPUT/*.zarr" --recursive "$OUTPUT" s3://"$S3_TARGET"
-        gsutil cp "$OUTPUT/vanderbilt/$TILES_BASE/*.ome.tif*" gs://vitessce-data/$TILES_BASE
+        aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --exclude "$OUTPUT/*.zarr/*" --recursive "$OUTPUT" s3://"$S3_TARGET"
+        gsutil -m cp -r "$OUTPUT/vanderbilt/vanderbilt.images" gs://"$S3_TARGET/vanderbilt/vanderbilt.images"
```

Comment:
```suggestion
        gsutil -m cp -r "$OUTPUT/vanderbilt/vanderbilt.images" "gs://$S3_TARGET/vanderbilt/vanderbilt.images"
```
No change in behavior, but quote the whole string for consistency.

(Perhaps rename S3_TARGET? Not a big deal.)

Reply:
Done. I'll open an issue to rename S3_TARGET.